test/lib/specHelperSpec.js

```diff
@@ -63,40 +67,25 @@ describe("Fluxxed up test helpers", function() {
     })
   })
 
-  describe("Component Helpers", function() {
-    var node, domNode, context
-
-    beforeEach(()=> {
-      //todo either expose a method for this. OR -- use old school way.
-      var div = document.createElement('div');
-      node = React.render(<TestComponent/>, div)
-      domNode = $(node.getDOMNode())
+  describe("Test Rig", function() {
+    var rig = new TestRig()
+    beforeEach(() => {
+      rig.boltOn(<TestComponent/>)
     })
-
-    afterEach(()=> {
-      React.unmountComponentAtNode(React.findDOMNode(node).parentNode);
+    afterEach(() => {
+      rig.boltOff()
     })
 
     it("renders", ()=> {
-      expect(domNode.find('h1').text()).to.equal('Test Component')
+      expect(rig.domNode.find('h1').text()).to.equal('Test Component')
```

Comment:
@BJK This works, as does `expect(rig.domNode.find('h1')).to.have.text('Test Component')`. Should we pick a style and stick with it?